modules/csharp_api/csharp/core/compiled_model.cs

```diff
@@ -0,0 +1,361 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OpenVinoSharp
+{
+    /// <summary>
+    /// This class represents a compiled model.
+    /// </summary>
+    /// <ingroup>ov_runtime_c#_api</ingroup>
+    /// <remarks>
+    /// A model is compiled by a specific device by applying multiple optimization 
+    /// transformations, then mapping to compute kernels.
+    /// </remarks>
+    public class CompiledModel : IDisposable
+    {
+        /// <summary>
+        /// [private]CompiledModel class pointer.
+        /// </summary>
+        private IntPtr m_ptr;
+        /// <summary>
+        /// [private]CompiledModel class pointer.
```

Comment:
Thank you very much for your valuable feedback! Because we call the C API in C #, pointer data needs to be passed between different methods, so different classes will obtain each other's pointers. Therefore, when encapsulating the interface, we need to expose the pointers．